sklearn/ensemble/iforest.py

```diff
@@ -92,7 +92,7 @@ class IsolationForest(BaseBagging, OutlierMixin):
         by `np.random`.
 
     verbose : int, optional (default=0)
-        Controls the verbosity of the tree building process.
```

Comment:
I think this should be reverted.

Reply:
> I think this should be reverted.

why?

Reply:
@jnothman I think ``IsolationForest`` is different from other classes in the PR. Parameter ``verbose`` will only influence the training process(``fit``), but will not influence the predict process(``predict``). (For other classes in the PR, parameter ``verbose`` will influence both training process and predicting process.) So I think the original statement here is suitable.

Reply:
I can confirm ``verbose`` does not change the output of ``predict`` for ``IsolationForest``.  I'll revert it.